### DISCUSSION TITLE
Software to create timelapse

### DISCUSSION THREAD

**Discussion by moskovskiy82** [blakeblackshear/frigate#6524] [General]
Any ideas are welcomed on a software (OS or free preferably) which can with minimum hassle create timelapsed videos from frigate folders?

**Comment by blakeblackshear** [comment]
I would use ffmpeg: https://medium.com/@sekhar.rahul/creating-a-time-lapse-video-on-the-command-line-with-ffmpeg-1a7566caf877

**Comment by moskovskiy82** [comment]
How to feed the input considering frigate path structure of date/hour/cam?

**Comment by blakeblackshear** [comment]
There is an API endpoint. See here for an example: https://github.com/blakeblackshear/frigate/issues/2369#issuecomment-1398023365

**Comment by uSlackr** [comment]
Any chance of this becoming a frigate feature?

**Reply by NickM-27** [reply]
This has already been implemented for 0.13

**Reply by hughc-hub** [reply]
thank you for the feature! May I ask if it would be possible to create a new feature to create automatic Timelapse's per week or month and not doing it manually?

thanks!

**Reply by NickM-27** [reply]
You can use the api and the frigate integration has a built in service for creating time lapses or normal exports.

**Reply by hughc-hub** [reply]
thank you for answering. I wish I have the knowledge to use the API for that.. I don´t have any idea to do that :(
